packages/sdk/src/fetcher/index.ts

```diff
@@ -91,21 +89,16 @@ class FullFetcher implements IFullCarrotFetcher {
         return useSubgraph
             ? SubgraphFetcher.fetchKPITokens({
                   provider,
-                  ipfsGatewayURL,
                   addresses,
-                  searchQuery,
```

Comment:
why are we removing the search directly on subgraph?

Reply:
I think with the separation between fetching and resolving we need to find another way to implement the search functionality. For now with both the subgraph and onchain fetcher we'll do that from the frontend.